### PR TITLE
cb250 implement money destination step

### PR DESCRIPTION
**Pull request by novadeveloper14**

### CODE REVIEW

**Review by Brandonz07 — CHANGES_REQUESTED** [review]
You need to create the mock, as shown in the image

image

File: src/pages/add-prospect/steps/MoneyDestination/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import { useMediaQuery } from "@inubekit/hooks";
+import { useEffect } from "react";
```
Comment (Brandonz07) [inline src/pages/add-prospect/steps/MoneyDestination/index.tsx:3]: Adjust imports, unify useState and useEffect

File: src/pages/add-prospect/steps/MoneyDestination/interface.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Stack } from "@inubekit/stack";
+
+import { IMoneyDestination } from "@src/services/types";
```
Comment (Brandonz07) [inline src/pages/add-prospect/steps/MoneyDestination/interface.tsx:3]: only services/types

File: src/components/cards/MoneyDestinationCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { Grid } from "@inubekit/grid";
+import { Text } from "@inubekit/text";
+import { Icon } from "@inubekit/icon";
+import { Stack } from "@inubekit/stack";
+import * as Icons from "react-icons/md";
+import { IconType } from "react-icons";
```
Comment (Brandonz07) [inline src/components/cards/MoneyDestinationCard/index.tsx:6]: Adjust imports

**Review by carin91 — APPROVED** [review]

**Pull request merged**